.github/workflows/ci.yaml

```diff
@@ -62,22 +62,6 @@ jobs:
               - 'charts/ingress-nginx/**/*'
               - 'NGINX_BASE'
 
-
-  security:
```

Comment:
gosec is failing to build/get with v1.21.1, but I have checked with golangci-lint and "all is fine" (ish) as we didn't cherrypicked the changes made on main to update the lint problems.

I will leave this way now to move forward on this release